docs/android/cli-tool.md

```diff
@@ -0,0 +1,15 @@
+# CLI Tool Instructions - Android
+
+## 1. Gradle Setup
+Follow [Gradle Instructions](gradle.md) to setup build environment
+
+
+## 2. Clean Build Directory and Run .jar
```

Comment:
`integration-cli` is a CLI on top of Gradle which is automating integration with PSPlugin. 
Tool is getting your regular gradle build command as input. 
*Please use flag `--stacktrace` together with gradle command.*

Also we're recommending clean your project before integration start.

**Example:**  
If you are building project with `./gradlew assembleRelease` your `integration-cli` command will look like:
```
rm -rf ./**/build
./gradlew clean
java -jar integration-cli-{{ android_release() }}.jar "./gradlew assembleRelease --stacktrace"
```